src/components/CreateVmWizard/steps/BasicSettings.js

```diff
@@ -309,11 +309,28 @@ class BasicSettings extends React.Component {
         changes.initSshKeys = template.getIn(['cloudInit', 'sshAuthorizedKeys'])
         changes.initTimezone = template.getIn(['cloudInit', 'timezone'])
         changes.initCustomScript = template.getIn(['cloudInit', 'customScript'])
+
+        if (changes.initTimezone && isOsWindows(changes.operatingSystemId, this.props.operatingSystems)) {
+          // Configure Timezone checkbox should be checked if template's timezone set
+          changes.enableInitTimezone = true
+          changes.lastInitTimezone = changes.initTimezone // select template's timezone in the Timezone drop down
+        } else {
+          changes.enableInitTimezone = false
+          changes.initTimezone = ''
+          // select the same default GMT sysprep timezone as in Admin Portal
+          changes.lastInitTimezone = timezones.find(timezone => timezone.value.startsWith('(GMT) Greenwich')).id
```

Comment:
This probably should be the default timezone value configured to the system, accessible in redux ... `state.config.get('defaultGeneralTimezone')` or `state.config.get('defaultWindowsTimezone')`

Reply:
As mentioned in my previous comment, they are not applicable here (different values, not what we want here).

Reply:
The default GMT sysprep timezone is not the same value as for the VM default Windows Timezone. Not sure why though, but that's the behaviour on backend and we should do the same here.